chain/chain/src/state_snapshot_actor.rs

```diff
@@ -21,48 +25,70 @@ impl StateSnapshotActor {
 }
 
 impl actix::Actor for StateSnapshotActor {
-    type Context = actix::Context<Self>;
+    type Context = Context<Self>;
 }
 
 #[derive(actix::Message, Debug)]
 #[rtype(result = "()")]
-struct MakeSnapshotRequest {
+struct DeleteSnapshotRequest {
+    /// Optionally send request to create a new snapshot after deleting any existing snapshots.
+    create_snapshot_request: Option<CreateSnapshotRequest>,
```

Comment:
Delete snapshot request is internal to chain/chain/src/state_snapshot_actor.rs and takes an optional create_snapshot_request